internal/ioutil_wrapper.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"io/ioutil"
+)
+
+// ReadFile is a wrapper to read a file. It is meant for internal use for testing.
+func ReadFile(filename string) ([]byte, error) {
```

Comment:
Consider moving this func to package stackdriver_test and making it an unexported func instead. Otherwise it may cause issues when import path `"contrib.go.opencensus.io/exporter/stackdriver/internal"` transformed.

Reply:
I have moved internal/ioutil_wrapper.go to util_test.go. 
What I don't like about the name util_test.go is that it implies that it contains some test (suffix _test is reserved for test files). If I just use util.go it doesn't allow me to use stackdriver_test package.

Reply:
Maybe test_helpers? e.g https://github.com/open-telemetry/opentelemetry-service/blob/master/config/test_helpers.go

Reply:
test_helpers.go is part of config package and so is config_test.go in the above example. They are all in the same package. 
In a single directory you can only have one package (XXX). You can have test package (XXX_test) but only for <file>_test.go. XXX_test can only use exported methods/fields from package XXX.
So the options are limited AFAIK.
1. internal package option (earlier commit)
2. put util in *_test.go file (this commit).

Reply:
Got it, I didn't realize XXX_test package can only contain *_test.go files. In this case I'd suggest going with util_test.go for now.